src/test/bin/quic_gtest.cpp

```diff
@@ -13,9 +13,7 @@
 bool TestingKernelMode = false;
 bool PrivateTestLibrary = false;
 bool UseDuoNic = false;
-#if defined(QUIC_USE_RAW_DATAPATH) && defined(QUIC_API_ENABLE_PREVIEW_FEATURES)
```

Comment:
Keep the `#if defined(QUIC_API_ENABLE_PREVIEW_FEATURES)`?

Reply:
This ifdef just wraps UseQTIP definition. No need to conditionally build

Reply:
It was also protecting usage of `QUIC_EXECUTION_CONFIG_FLAG_QTIP` below. The goal is to eventually remove QTIP, so I'd rather have it isolated as much as possible.